rhobs/recording/release_service_availability_recording_rules.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-release-service-availability
+  labels:
+    tenant: rhtap
+spec:
+
+  groups:
+    - name: release_service_general_availability
+      interval: 1m
+      rules:
+        - record: konflux_up
+          expr: |
+            label_replace(
+              avg(release_service_check_gauge) without(Name) OR
```

Comment:
As we mention the label `Name` here, I think we should also have it inside the input series to verify things are aggregated as expected.

Reply:
True. As I am artificially adding the "release"  one, it duplicates the Name - hence I had to remove it, before re-adding it. But I forgot to update with the last operation. Fixed!

Reply:
The test now shows that it's being added. I think we should also have it in some of the inputs to test how we handle also cases in which it's present in the first place in `release_service_check_gauge`.